About.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>  
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <header>
+        <div id="black-header " class="display_desktop"> 
+            <ul class="menu-black">
+               
+                <li><img src="images/facebook.png"></li>
+                <li><img src="images/twitter.png"></li>
+                <li>Logout</li>
+                <li>Mypage</li>
+                <li>Korean</li>
+            </ul>
+            </div>
+        <nav class="display_desktop navigation">
+            
+            <ul class="desktop_menu">
+                <li> <img src="images/logo.png" class="logo"></li>
+                <li ><a href="About.html" class="about-link">About</a></li>
```

Comment:
- [  ] As indicated by the previous reviewer, The current page here is the About page, hence, it does not make sense to have this anchor tag link to the same page. The suggestion is to edit the text from `About` to `Home`

### View:
![image](https://github.com/umarahgul/capstone-1/assets/50990925/381ab963-fcfd-44f9-a58c-28f100903c4c)